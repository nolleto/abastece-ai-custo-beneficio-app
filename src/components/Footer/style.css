.footer {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin-top: 50px;
  font-size: 12px;
  text-transform: uppercase;
}

.red {
  color: #E91E63;
}

.line {
  width: 80%;
  background-color: gray;
  height: 1px;
}
