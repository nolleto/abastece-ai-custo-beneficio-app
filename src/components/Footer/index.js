import React from 'react';
import './style.css';

const Footer = () => (
  <footer className="footer">
    <div className='line' />
    <p>
      Made with <span className="red">❤︎</span> by <a href="https://twitter.com/FelipeNolleto">nolleto</a>.
      Find this on <a href="https://github.com/nolleto/mineires-translator-app">github</a>
    </p>
  </footer>
);

export default Footer;
