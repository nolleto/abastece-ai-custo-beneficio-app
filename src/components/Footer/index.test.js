import React from 'react';
import Footer from './index';
import { shallow } from 'enzyme';

describe('Footer component', () => {
  it('matches snapshot', () => {
    const snapshot = shallow(<Footer />);

    expect(snapshot).toMatchSnapshot();
  });
});
