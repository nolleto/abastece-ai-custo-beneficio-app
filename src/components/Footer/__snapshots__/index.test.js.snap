// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="line"
  />
  <p>
    Made with 
    <span
      className="red"
    >
      ❤︎
    </span>
     by 
    <a
      href="https://twitter.com/FelipeNolleto"
    >
      nolleto
    </a>
    . Find this on 
    <a
      href="https://github.com/nolleto/abastece-ai-custo-beneficio-app"
    >
      github
    </a>
  </p>
</footer>
`;
